pom.xml

```diff
@@ -120,12 +120,13 @@
                 <groupId>log4j</groupId>
                 <artifactId>log4j</artifactId>
                 <version>1.2.17</version>
+                <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-log4j12</artifactId>
                 <!-- make sure this matches the slf4j version in the dropwizard bom -->
-                <version>1.7.26</version>
+                <version>1.7.30</version>
```

Comment:
Its a bit unfortunate the dropwizard BOM does not declare this dependency and there is no way we can access the `slf4j.version` property from there right?

Reply:
ok i did this